firmware/application/recent_entries.hpp

```diff
@@ -179,6 +179,24 @@ class RecentEntriesTable : public Widget {
 				}
 			}
 		}
+		else if( event == ui::KeyEvent::Up ) {
```

Comment:
what is that section doing? does it affect cursor navigation?

Reply:
Yes, just a small usability update that allows your to use the buttons to go up and down through the list. I found it jarring to be able to use it to get to the table, but not go through the entries. This will let you scroll through the list with them, and when it hits the top or bottom record, it will default back to the focus handling logic for the next widget.

Reply:
OK thanks, I am a bit afraid of this global things :) I will try to test it in other instances